### PR TITLE
Update tests baselines

### PR DESCRIPTION
**Pull request by vzarytovskii**
Updated majority of the tests, some of them are failing (produce ERROR and UNEXPECTED nodes), I suggest they're handled  case-by-case.

Also, added test for https://github.com/Nsidorenco/tree-sitter-fsharp/issues/4

### CODE REVIEW

File: corpus/compiler_directive.txt [hunk 1]
```diff
@@ -1,27 +1,26 @@
...
-    (string)
-    (string)))
+--------------------------------------------------------------------------------
 
+(ERROR
+  (UNEXPECTED 'r')
+  (UNEXPECTED 'h')
+  (UNEXPECTED '"')
+  (UNEXPECTED 'e')
+  (UNEXPECTED '"'))
```
Comment (vzarytovskii) [inline corpus/compiler_directive.txt:26]: That looks like regression, but it's there forever it seems. I can look at it.

**Comment by Nsidorenco** [comment]
Super nice, thank you for doing this @vzarytovskii!

**Comment by vzarytovskii** [comment]
> Super nice, thank you for doing this @vzarytovskii!

Thanks for making the grammar! Happy to contribute. I have several fixes in mind already.

File: corpus/compiler_directive.txt (hunk 1 above)
Comment (Nsidorenco) [reply corpus/compiler_directive.txt:26]: Yeah, I think this is a relic from before modules and expression were added to the grammar

**Comment by Nsidorenco** [comment]
Great to hear! I'll go ahead and merge this so we have a better foundation for improving the grammar. 
We can get around to the failing cases when we fix that part.

Just as a heads-up, in #8 im trying to improve how the external parser deals with expressions and list/records, if you stumble upon some of the issues relating to CE expression/record definitions this might fix that.

**Pull request merged**